styles.css

```diff
@@ -0,0 +1,586 @@
+  height: 32px;
+  display: flex;
+  justify-content: center;
+}
+
+h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 40px;
+  line-height: 108%;
+  text-align: center;
+  display: flex;
+  color: #172b4d;
+  justify-content: center;
+}
+
+section p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #172b4d;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  text-align: center;
+  margin-top: -10%;
+}
+
+section h3 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 117px;
+  height: 24px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.green {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 40px;
+  line-height: 130%;
+  text-align: center;
+  display: flex;
+  color: #36b37e;
+  justify-content: center;
+}
+
+h1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 40px;
+  line-height: 130%;
+  text-align: center;
+  display: flex;
+  color: #172b4d;
+  justify-content: center;
+}
+
+.menu_main .toggler {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: auto;
+  z-index: 4;
+  cursor: pointer;
+  width: 50px;
+  height: 50px;
+  opacity: 0;
+}
+
+.menu_main .humbuger {
+  position: absolute;
+  right: 0;
+  top: 0;
+  z-index: 3;
+  width: 60px;
+  height: 60px;
+  padding: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+/* .humbuger line */
+
+.menu_main .humbuger > div {
+  position: relative;
+  width: 100%;
+  height: 1px;
+  background-color: blue;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: all 0.4s ease;
+}
+
+/* Top and bottom line */
+.menu_main .humbuger > div::after,
+.menu_main .humbuger > div::before {
+  content: "";
+  position: absolute;
+  top: 0.25em;
+  left: 0;
+  width: 1em;
+  height: 0.125em;
+  border-top: 0.375em double #000;
+  border-bottom: 0.125em solid #000;
+}
+
+/* moves lin down */
+.menu_main .humbuger > div::after {
+  top: 10px;
+  width: 100%;
+}
+
+.menu_main .menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  margin: auto;
+  margin-left: 10%;
+  margin-top: 2%;
+  height: 100%;
+  width: 100%;
+  border-radius: 0;
+  overflow: hidden;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  visibility: unset;
+}
+
+.overlay ::before {
+  content: "";
+  position: absolute;
+  top: 1em;
+  left: 300;
+  width: 1em;
+  height: 0.125em;
+  border-top: 0.375em double #000;
+  border-bottom: 0.125em solid #000;
+}
+
+.overlay ul {
+  visibility: hidden;
+}
+
+header img {
+  display: flex;
+  align-content: flex-end;
+  margin-left: 300px;
+  margin-top: 0%;
+}
+
+.headline {
+  background-image: url(icons/background.png);
+  background-position: initial;
+  background-repeat: no-repeat;
+  background-size: contain;
+  gap: 12px;
+  border: none;
+}
```

Comment:
- [ ] Kindly adjust the background image to match the Figma design as suggested by the previous reviewer. ✍️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/tsigewossne/portfolio/assets/17243325/50b2b422-ad0c-4760-a853-a96ca3102fab" width="300">|<img src="https://github.com/tsigewossne/portfolio/assets/17243325/5dd3ced0-2274-4ebf-9e45-1bd8213262f2" width="300">|